pkg/ingester/transfer.go

```diff
@@ -284,7 +290,8 @@ func (i *Ingester) TransferTSDB(stream client.Ingester_TransferTSDBServer) error
 		receivedFiles.Add(float64(filesXfer))
 		level.Error(util.Logger).Log("msg", "Total files xfer", "from_ingester", fromIngesterID, "num", filesXfer)
```

Comment:
Unrelated from this changes, but I believe this should be an `Info`

Reply:
yup fixed.